commit 637ba66c486ff96aa2b60d8036cf6a63c84f6f10 (refs/changes/69/4669/4)
Author: Jared Morrow <jm@joyent.com>
Date:   2018-08-27T15:44:03-06:00 (1 year, 1 month ago)
    
    MANTA-1387 libmanta should log and ignore BucketVersionError
    Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
    Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

diff --git a/package.json b/package.json
index 0b04a23..ec2a59a 100644
--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
         "kang": "1.1.0",
         "keep-alive-agent": "0.0.1",
         "keyapi": "git+https://github.com/joyent/keyapi.git#e14b3d58",
-        "libmanta": "git+https://github.com/joyent/node-libmanta.git#master",
+        "libmanta": "git+https://github.com/joyent/node-libmanta.git#v1.1.1",
         "libuuid": "0.1.2",
         "lru-cache": "2.3.1",
         "lstream": "0.0.4",
